tracer/test/Datadog.Trace.Tools.Runner.ArtifactTests/CheckCommandTests.cs

```diff
@@ -22,4 +23,20 @@ public void Help()
         helper.StandardOutput.Should().Contain("dd-trace check [command] [options]");
         helper.ErrorOutput.Should().Contain("Required command was not provided.");
     }
+
+    [SkippableFact]
+    public void MultipleArguments()
+    {
+        // Currently we can only test this one Windows.
+        // The only command that accepts a space is "check iis" and it's not available on Linux.
```

Comment:
Can't we have spaces in `dd-dotnet run`, if there's a space in the path to the executable?

Reply:
Only the `check` command runs on Linux